src/io/stdin.rs

```diff
@@ -218,3 +255,13 @@ cfg_if! {
         }
     }
 }
+
+impl Read for StdinLock<'_> {
+    fn poll_read(
+        self: Pin<&mut Self>,
+        _cx: &mut Context<'_>,
+        _buf: &mut [u8],
+    ) -> Poll<io::Result<usize>> {
+        unimplemented!()
```

Comment:
Is this better to mimic the Stdin implementation, or?

Reply:
This should forward the inner read impl so we can actually read the bytes from Stdin. So, yes! :D

Reply:
thanks !

Reply:
This needs to forward to the inner `Read` impl.